pages/conference/young-inspirers.vue

```diff
@@ -1,266 +1,236 @@
-                        v-for="(speaker, index) in speechData.speakers"
-                        :key="`young_inspirers_tab_speaker_${index}`"
-                        class="mb-4 whitespace-pre-line"
+                        v-for="(speech, speechIdx) in speechInfo.speeches"
+                        :key="`speech_${speechIdx}`"
+                        class="speech__box__wrapper"
                     >
-                        <div class="whitespace-pre-line">
-                            <p class="tabParagraphTitle">
-                                {{ speaker.name }}
-                            </p>
-                            <p class="tabParagraph">{{ speaker.bio }}</p>
+                        <div class="speechBox">
+                            <div>
+                                <div class="speechBox__avater__wrapper">
+                                    <img
+                                        v-for="(speakerAvater, i) in tagToPhoto[
+                                            speech.tag
+                                        ]"
+                                        :key="`speech_info_${speechIdx}_photo_${i}`"
+                                        :src="speakerAvater"
+                                        :alt="tagToSpeaker[speech.tag][i]"
+                                        class="speechBox__avater"
+                                    />
+                                </div>
+                                <p class="speechBox__time">
+                                    {{ speech.speechtime }}
+                                    <img
+                                        class="speechBox__langIcon"
+                                        :src="tagToLangIcon[speech.tag]"
+                                    />
+                                </p>
+                                <p class="speechBox__title">
+                                    {{ speech.title }}
+                                </p>
+                                <div
+                                    class="speechBox__speaker"
+                                    @click="
+                                        popupShow(
+                                            `${speechDayIdx}-${speechIdx}`,
+                                        )
+                                    "
+                                >
+                                    <span
+                                        v-for="(speaker, i) in tagToSpeaker[
+                                            speech.tag
+                                        ]"
+                                        :key="`speech_info_${speechIdx}_speaker_${i}`"
+                                    >
+                                        {{ speaker }}
+                                    </span>
+                                </div>
+                                <p class="speechBox__description">
+                                    {{ speech.full_description }}
```

Comment:
The description should be truncated if it's too long. The following is the snapshot of figma.

<img width="344" alt="image" src="https://user-images.githubusercontent.com/24987826/177976295-1bebc876-e276-491d-a684-3b57a05b0f3d.png">

Reply:
spec deleted